maxim/logger/fireworks/utils.py

```
@@ -0,0 +1,280 @@
+                                    generation.add_attachment(UrlAttachment(
+                                        url=image_url,
+                                        name="User Image",
+                                    ))
+        except Exception as e:
+            scribe().warning(f"[MaximSDK] Error adding image attachments: {e}")
+            
+    @staticmethod
+    def apply_tags(
+        generation: Optional[Generation], 
+        trace: Optional[Trace], 
+        generation_tags: Optional[Dict[str, str]], 
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)